packages/webawesome/docs/.eleventy.js

```diff
@@ -94,6 +94,16 @@ export default async function (eleventyConfig) {
     flashes: '',
   });
 
+  // Site metadata for social sharing (Open Graph, canonical URLs, etc.)
+  const site = {
+    url: 'https://webawesome.com',
+    name: 'Web Awesome',
+    description: 'Build better with Web Awesome, the open source library of web components from Font Awesome.',
+    image: 'https://webawesome.com/assets/images/open-graph/default.png',
+  };
+
+  eleventyConfig.addGlobalData('site', site);
```

Comment:
```suggestion
  eleventyConfig.addGlobalData('metadata', site);
```

I'd suggest changing the key to something a bit less likely to conflict. Perhaps `metadata` or `websiteData` or similar.